boost-maven-plugin/src/main/java/io/openliberty/boost/liberty/LibertyPackageMojo.java

```diff
@@ -104,13 +124,15 @@ public void execute() throws MojoExecutionException {
     private void copySpringBootUberJar() throws MojoExecutionException {
         try {
             if (!SpringBootUtil.copySpringBootUberJar(project.getArtifact().getFile())) {
-                File springJar = new File(SpringBootUtil.getSpringBootUberJarPath(project.getArtifact().getFile()));
-                if (springJar.exists() && springJar.isFile()) {
+                File springJar = new File(
+                        SpringBootUtil.getBoostedSpringBootUberJarPath(project.getArtifact().getFile()));
+                if (springJar.exists() && springJar.isFile()
+                        && net.wasdev.wlp.common.plugins.util.SpringBootUtil.isSpringBootUberJar(springJar)) {
```

Comment:
The isSpringBootUberJar checks for exists() and isFile(), so these checks can be removed here.

Reply:
Done.